src/workers/pool-number-refresh.js

```diff
@@ -0,0 +1,94 @@
+import Twilio from 'twilio'
+// import { r } from '../server/models'
+
+/**
+ * Get all numbers contacted in last 30 days.
+ */
+const getRecentContactNumbers = () => {
+  // TODO: replace this with database call after thinky to knex switch
```

Comment:
Probably, but since we have the numbers in the database, that should be faster and more reliable.